torch/csrc/autograd/saved_variable.h

```diff
@@ -46,6 +46,10 @@ class TORCH_API SavedVariable {
   }
 
  private:
+  // The variable to save.
```

Comment:
The doc here should mention the link between the content of this and the boolean saved_original below.